src/dmd/json.d

```diff
@@ -793,19 +793,62 @@ public:
         jsonProperties(d);
         objectEnd();
     }
+
+    private void generateBuildInfo()
+    {
+        objectStart();
+        property("kind", "buildInfo");
+        property("binary", global.params.argv0);
+        property("version", global._version);
+        property("config", global.inifilename ? global.inifilename : null);
+        if (global.params.lib) {
+            property("library", global.params.libname);
+        }
+        objectEnd();
+    }
+    private void generateSemantics()
+    {
+        objectStart();
+        property("kind", "semantics");
+        propertyStart("modules");
+        arrayStart();
+        foreach (m; Module.amodules)
+        {
+            objectStart();
+            if(m.md)
+                property("name", m.md.toChars());
+            property("file", m.srcfile.toChars());
+            propertyBool("isRoot", m.isRoot());
+            if(m.contentImportedFiles.dim > 0)
+            {
+                propertyStart("contentImports");
+                arrayStart();
+                foreach (file; m.contentImportedFiles)
+                {
+                    value(file);
```

Comment:
BUG! missing separaters in generated JSON
"contentImports" : [
"/tmp/myfile.txt""/tmp/myfile.txt"
    ]